Kitodo/src/main/java/org/kitodo/services/workflow/WorkflowService.java

```diff
@@ -0,0 +1,726 @@
+                if (this.user != null) {
+                    task.setProcessingUser(this.user);
+                }
+            }
+        }
+        return task;
+    }
+
+    /**
+     * Change Task status down.
+     *
+     * @param task
+     *            to change status down
+     * @return updated task
+     */
+    public Task setTaskStatusDown(Task task) {
+        task.setEditTypeEnum(TaskEditType.ADMIN);
+        task.setProcessingTime(new Date());
+        if (this.user != null) {
+            task.setProcessingUser(this.user);
+        }
+        return serviceManager.getTaskService().setProcessingStatusDown(task);
+    }
+
+    /**
+     * Change Task status up for list of tasks assigned to given Process.
+     * 
+     * @param process
+     *            object
+     */
+    public void setTasksStatusUp(Process process) throws DataException, IOException {
+        List<Task> tasks = process.getTasks();
+
+        for (Task task : tasks) {
+            if (!task.getProcessingStatus().equals(TaskStatus.DONE.getValue())) {
+                task.setProcessingStatus(task.getProcessingStatus() + 1);
+                task.setEditType(TaskEditType.ADMIN.getValue());
+                if (task.getProcessingStatus().equals(TaskStatus.DONE.getValue())) {
+                    close(task);
+                } else {
+                    if (this.user != null) {
+                        task.setProcessingUser(this.user);
+                        serviceManager.getTaskService().save(task);
+                    }
+                }
+                break;
+            }
+        }
+    }
+
+    /**
+     * Change Task status down for list of tasks assigned to given Process.
+     * 
+     * @param process
+     *            object
+     */
+    public void setTasksStatusDown(Process process) throws DataException {
+        List<Task> tasks = new ArrayList<>(process.getTasks());
+        Collections.reverse(tasks);
+
+        for (Task task : tasks) {
+            //TODO: check if this behaviour is correct
+            if (process.getTasks().get(0) != task && task.getProcessingStatusEnum() != TaskStatus.LOCKED) {
+                task.setEditTypeEnum(TaskEditType.ADMIN);
+                task.setProcessingTime(new Date());
+                if (this.user != null) {
+                    task.setProcessingUser(this.user);
+                }
+                task = serviceManager.getTaskService().setProcessingStatusDown(task);
+                serviceManager.getTaskService().save(task);
+                break;
+            }
+        }
+    }
+
+    /**
+     * Not sure.
+     *
+     * @param task
+     *            object
+     * @return closed Task
+     */
+    public Task closeTaskByUser(Task task) throws DataException, IOException {
+
+        // if step allows writing of images, then count all images here
+        if (task.isTypeImagesWrite()) {
+            try {
+                // this.mySchritt.getProzess().setSortHelperImages(
```

Comment:
Please remove out commented code. This code is commented out even in 2.x branch.

Reply:
Done.